mix.exs

```diff
@@ -11,7 +11,7 @@ defmodule Elixium.Mixfile do
       deps: deps(),
```

Comment:
Might need to start ```:logger``` under the extra apps for the tests

Reply:
Weird... I'll take a look. This wasn't failing before, I may need to declare it in the Pico library

Reply:
Added logger as an extra application and it fixed the tests